### ISSUE TITLE
Support for Flutter Web

### ISSUE THREAD

**Issue by srix55** [eBay/flutter_glove_box#80]
Currently this package is not compatible with web platform. Please consider adding support for flutter web. Thanks.

![image](https://user-images.githubusercontent.com/6329365/99029154-c0363700-2597-11eb-9225-6edec3041736.png)

**Comment by fatherOfLegends** [comment]
@srix55 This is odd because the package is exclusively for testing. As a dev dependency it should work just fine in a flutter app that targets the web. Do you have a failing example or is it just the static analysis rule on pub.dev?

**Comment by srix55** [comment]
@fatherOfLegends You are right. I haven't had any problems while running my tests. Just that I noticed that the package would not show up when I filtered by web in pub.dev. 
I guess the expectation is to have this show up as web compatible in pub as many users would think this won't work for apps targeting web, or might not see this package at all when filtering by platforms.

**Comment by mZadorskii** [comment]
dar io does not support web

**Comment by fatherOfLegends** [comment]
@srix55 I agree with the discovery issue but I cannot think of a way around it. Do you have any ideas regarding how we could creatively resolve this?

**Comment by fatherOfLegends** [comment]
Here is an issue discussing this at length: https://github.com/dart-lang/pana/issues/720

**Comment by srix55** [comment]
@fatherOfLegends Thanks for the update. I had a look at the flutter_test package code for half an hour and I am still pretty clueless about the fix...
